### PR TITLE
Adds proper support for light/dark themes

### PR DESCRIPTION
**Pull request by markusingierd**
This pull request adds proper support for light/dark themes, and makes use of the application's colors through the theme, instead of directly to better support dynamic switching.

Note: Not merged with master, but only merged files of interest are theme refactoring in most of the screen classes

### CODE REVIEW

**Review by Anarox1111 — CHANGES_REQUESTED** [review]
Must be applied to `master`'s structure before merging.

Code looks good other than that!

File: app/src/main/java/no/usn/mob3000/App.kt [hunk 1]
```diff
@@ -563,9 +561,13 @@ fun TopNavbar(
     topBarActions: @Composable (RowScope.() -> Unit) = {},
 ) {
     TopAppBar(
-        title = { if (showTitle) Text(stringResource(currentScreen.title)) },
+        title = { if (showTitle)
+            Text(stringResource(currentScreen.title),
+                color = MaterialTheme.colorScheme.onSurface
+            ) },
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/App.kt:567]: Remember indentation.

File: app/src/main/java/no/usn/mob3000/App.kt [hunk 2]
```diff
@@ -604,14 +606,15 @@ fun TopNavbar(
  * @author frigvid
  * @created 2024-10-02
  */
+
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/App.kt:609]: Remember spacing. Do not introduce previously non-existent spacing issues, please.

File: app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt [hunk 3]
```diff
@@ -2,16 +2,16 @@ package no.usn.mob3000.ui.components
...
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.window.Dialog
 import no.usn.mob3000.R
-import no.usn.mob3000.ui.theme.DefaultButton
+
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt:14]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt [hunk 4]
```diff
@@ -34,9 +34,7 @@ fun DangerousActionDialogue(
     onDismiss: () -> Unit,
     confirmText: String = stringResource(R.string.component_dialogue_dangerous_action_button_confirm),
     dismissText: String = stringResource(R.string.component_dialogue_dangerous_action_button_dismiss),
-    colorContainer: Color = MaterialTheme.colorScheme.surface,
-    colorButtonDismiss: Color = DefaultButton,
-    colorButtonConfirm: Color = Color.Red,
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt:39]: Incorrect change, and somewhat improperly applied. This needs to be rolled back. Remember that if a function is using input parameters, much like the dangerous action dialogue, there's often a reason for it. Now you've practically hard-coded the colors, which means that calls to it can no longer override default colors if necessary. In this case, you should have added the colors that would need to have been changed as the new default value of the parameter list and not _remove_ the parameters.

File: app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt [hunk 5]
```diff
@@ -66,15 +64,21 @@ fun DangerousActionDialogue(
...
                             .padding(end = 8.dp)
                     ) { Text(dismissText) }
 
                     Button(
                         onClick = onConfirm,
-                        colors = ButtonDefaults.buttonColors(colorButtonConfirm),
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = Color(0xFFFF0000),
+                            contentColor = Color.White
+                        ),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt:81]: This is now less flexible than before. Not only is it hard-coded, but it's also using hexadecimal color codes when a simple `Color.Red` would've sufficed as before. But, again, refer to [this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840946578).

File: app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt [hunk 6]
```diff
@@ -34,9 +34,7 @@ fun DangerousActionDialogue(
     onDismiss: () -> Unit,
     confirmText: String = stringResource(R.string.component_dialogue_dangerous_action_button_confirm),
     dismissText: String = stringResource(R.string.component_dialogue_dangerous_action_button_dismiss),
-    colorContainer: Color = MaterialTheme.colorScheme.surface,
-    colorButtonDismiss: Color = DefaultButton,
-    colorButtonConfirm: Color = Color.Red,
+
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt:37]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/SettingsScreen.kt [hunk 7]
```diff
@@ -127,7 +127,8 @@ fun SettingsScreen(
 
             Button(
                 onClick = onAdminDashboardClick,
-                colors = ButtonDefaults.buttonColors(DefaultButton),
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = MaterialTheme.colorScheme.primary),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/SettingsScreen.kt:131]: Remember indentation.

Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/SettingsScreen.kt:131]: Remember indentation. Specifically, the closing parenthesis should always be on a newline in this sort of use-case.

File: app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt [hunk 8]
```diff
@@ -90,18 +90,20 @@ fun LoginScreen(
...
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
                     keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Email)
                 )
 
                 OutlinedTextField(
                     value = inputPassword,
                     onValueChange = { inputPassword = it },
                     label = { Text(stringResource(R.string.auth_login_password)) },
-                    placeholder = { Text(stringResource(R.string.auth_login_password_placeholder)) },
+                    placeholder = { Text(stringResource(R.string.auth_login_password_placeholder), ) },
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt:101]: Incorrectly applied comma and space.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 9]
```diff
@@ -48,7 +49,7 @@ fun PlayScreen() {
             verticalArrangement = Arrangement.spacedBy(16.dp)
         ) {
 
-            Card(colors = CardDefaults.cardColors(containerColor = DefaultListItemBackground)) {
+            Card( colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primary)) {
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:52]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 10]
```diff
@@ -57,17 +58,17 @@ fun PlayScreen() {
                     Text(
                         text = stringResource(R.string.play_game_status),
                         fontSize = 18.sp,
-                        fontWeight = FontWeight.Bold
+                        fontWeight = FontWeight.Bold,
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:61]: Unnecessary change.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 11]
```diff
@@ -57,17 +58,17 @@ fun PlayScreen() {
...
                     )
 
                     Text(
                         text = gameStatus,
-                        fontSize = 16.sp
+                        fontSize = 16.sp,
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:66]: Unnecessary change.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 12]
```diff
@@ -76,7 +77,7 @@ fun PlayScreen() {
                     Text(
                         text = stringResource(R.string.play_statistics),
                         fontSize = 18.sp,
-                        fontWeight = FontWeight.Bold
+                        fontWeight = FontWeight.Bold,
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:80]: Unnecessary change.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 13]
```diff
@@ -57,17 +58,17 @@ fun PlayScreen() {
...
                     Text(
                         text = gameStatus,
-                        fontSize = 16.sp
+                        fontSize = 16.sp,
                     )
                 }
             }
 
-            Card(colors = CardDefaults.cardColors(containerColor = DefaultListItemBackground)) {
+            Card( colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primary)) {
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:71]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 14]
```diff
@@ -105,20 +106,17 @@ fun PlayScreen() {
             ) {
                 Button(
                     onClick = { /* TODO: Implement reset board logic */ },
-                    colors = ButtonDefaults.buttonColors(containerColor = DefaultButton),
-                    modifier = Modifier.weight(1f).padding(end = 8.dp)
+                    colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),                    modifier = Modifier.weight(1f).padding(end = 8.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:109]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 15]
```diff
@@ -105,20 +106,17 @@ fun PlayScreen() {
...
                 ) { Text(stringResource(R.string.play_reset_board)) }
 
                 Button(
                     onClick = { /* TODO: Implement undo move logic */ },
-                    colors = ButtonDefaults.buttonColors(containerColor = DefaultButton),
-                    modifier = Modifier.weight(1f).padding(horizontal = 4.dp)
+                    colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),                    modifier = Modifier.weight(1f).padding(horizontal = 4.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:114]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt [hunk 16]
```diff
@@ -105,20 +106,17 @@ fun PlayScreen() {
...
                 ) { Text(stringResource(R.string.play_undo_move)) }
 
                 Button(
                     onClick = { /* TODO: Implement switch to multiplayer logic */ },
-                    colors = ButtonDefaults.buttonColors(containerColor = DefaultButton),
-                    modifier = Modifier.weight(1f).padding(start = 8.dp)
+                    colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),                    modifier = Modifier.weight(1f).padding(start = 8.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt:119]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt [hunk 17]
```diff
@@ -54,8 +55,8 @@ fun CreateGroupScreen(
             modifier = Modifier.padding(innerPadding),
             bottomBar = {
                 BottomAppBar(
-                    containerColor = MaterialTheme.colorScheme.surface,
-                    contentColor = MaterialTheme.colorScheme.onSurface,
+                    containerColor = MaterialTheme.colorScheme.primary,
+                    contentColor = MaterialTheme.colorScheme.onPrimary,
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt:59]: These parameters should keep using `surface` and `onSurface`. The same `Color.kt` values can simply be applied to those parameters in the color schemes and then made use of.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt [hunk 18]
```diff
@@ -70,14 +71,14 @@ fun CreateGroupScreen(
                                 selectedOpenings = emptyList()
                                 /* TODO: onClearGroup() */
                             },
-                            colors = ButtonDefaults.buttonColors(DefaultButton),
+                            colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt:74]: Unnecessarily explicit, simply inputting the color as before is preferable.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt [hunk 19]
```diff
@@ -70,14 +71,14 @@ fun CreateGroupScreen(
...
                             modifier = Modifier.weight(1f).padding(end = 8.dp)
                         ) {
                             Text(stringResource(R.string.group_create_reset_all))
                         }
                         Button(
                             onClick = { /* TODO: onSaveGroup(title, description, selectedOpenings) */ },
-                            colors = ButtonDefaults.buttonColors(DefaultButton),
+                            colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt:81]: Unnecessarily explicit, simply inputting the color as before is preferable.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt [hunk 20]
```diff
@@ -76,15 +75,19 @@ fun GroupsScreen(
     Viewport(
         floatingActionButton = {
             FloatingActionButton(
-                containerColor = DefaultButton,
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt:79]: Unnecessary modifier applied. `FloatingActionButton` composables have shadows by default.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt [hunk 21]
```diff
@@ -130,8 +133,8 @@ fun GroupItem(group: Group) {
         modifier = Modifier
             .fillMaxWidth()
             .clickable { expanded = !expanded },
-        colors = CardDefaults.cardColors(containerColor = DefaultListItemBackground)
-    ) {
+        colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primary),
+        ) {
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt:137]: Remember indentation.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt [hunk 22]
```diff
@@ -130,8 +133,8 @@ fun GroupItem(group: Group) {
         modifier = Modifier
             .fillMaxWidth()
             .clickable { expanded = !expanded },
-        colors = CardDefaults.cardColors(containerColor = DefaultListItemBackground)
-    ) {
+        colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primary),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/GroupsScreen.kt:136]: Unnecessary trailing comma.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/CreateOpeningScreen.kt [hunk 23]
```diff
@@ -51,6 +53,7 @@ fun CreateOpeningScreen(
                 value = title,
                 onValueChange = { title = it },
                 label = { Text(stringResource(R.string.opening_create_prompt_title)) },
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground,),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/CreateOpeningScreen.kt:56]: Unnecessary internal trailing comma. There are no additional parameters inputted, and there shouldn't be trailing commas like this.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/CreateOpeningScreen.kt [hunk 24]
```diff
@@ -59,6 +62,7 @@ fun CreateOpeningScreen(
                 onValueChange = { description = it },
                 label = { Text(stringResource(R.string.opening_create_prompt_description)) },
                 modifier = Modifier.fillMaxWidth(),
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground,),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/CreateOpeningScreen.kt:65]: Unnecessary internal trailing comma. There are no additional parameters inputted, and there shouldn't be trailing commas like this.

File: app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/OpeningsScreen.kt [hunk 25]
```diff
@@ -78,15 +77,19 @@ fun OpeningsScreen(
     Viewport (
         floatingActionButton = {
             FloatingActionButton(
-                containerColor = DefaultButton,
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/OpeningsScreen.kt:81]: [See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).

File: app/src/main/java/no/usn/mob3000/ui/screens/info/docs/CreateDocumentationScreen.kt [hunk 26]
```diff
@@ -86,7 +89,10 @@ fun CreateDocumentationScreen(
             Button(
                 onClick = onSaveDocumentationClick,
                 modifier = Modifier.fillMaxWidth(),
-                colors = ButtonDefaults.buttonColors(DefaultButton)
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = MaterialTheme.colorScheme.primary, // Bruker primary fargen fra temaet
+                    contentColor = MaterialTheme.colorScheme.onPrimary // Bruker fargen for tekst (content) på primary
+                )
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/docs/CreateDocumentationScreen.kt:95]: If you are using AI, you need to APA7 reference the conversation and also have the conversation browsable for the teachers perusal. Preferably, do not use AI. Additionally, these comments serve no purpose, they literally just repeat what the code already says explicitly.

File: app/src/main/java/no/usn/mob3000/ui/screens/info/docs/DocumentationDetailsScreen.kt [hunk 27]
```diff
@@ -86,8 +85,7 @@ fun DocumentationDetailsScreen(
                     modifier = Modifier
                         .align(Alignment.BottomCenter)
                         .fillMaxWidth()
-                        .background(Color(0xFFEFEBE9))
-                        .padding(16.dp)
+                        .background(MaterialTheme.colorScheme.background)                        .padding(16.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/docs/DocumentationDetailsScreen.kt:88]: Remember spacing.

File: app/src/main/java/no/usn/mob3000/ui/screens/info/docs/DocumentationScreen.kt [hunk 28]
```diff
@@ -93,7 +92,8 @@ fun DocumentationScreen(
         floatingActionButton = {
             FloatingActionButton(
                 onClick = onCreateDocumentationClick,
-                containerColor = DefaultButton
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/docs/DocumentationScreen.kt:96]: [See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).

File: app/src/main/java/no/usn/mob3000/ui/screens/info/faq/FAQScreen.kt [hunk 29]
```diff
@@ -88,7 +90,8 @@ fun FAQScreen(
         floatingActionButton = {
             FloatingActionButton(
                 onClick = onCreateFAQClick,
-                containerColor = DefaultButton
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/faq/FAQScreen.kt:94]: [See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).

File: app/src/main/java/no/usn/mob3000/ui/screens/info/news/CreateNewsScreen.kt [hunk 30]
```diff
@@ -85,7 +88,7 @@ fun CreateNewsScreen(
             Button(
                 onClick = onSaveNewsClick,
                 modifier = Modifier.fillMaxWidth(),
-                colors = ButtonDefaults.buttonColors(DefaultButton)
+                colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/news/CreateNewsScreen.kt:91]: Unnecessarily explicit, simply inputting the color as before is preferable.

File: app/src/main/java/no/usn/mob3000/ui/screens/info/news/NewsScreen.kt [hunk 31]
```diff
@@ -100,7 +99,8 @@ fun NewsScreen(
         floatingActionButton = {
             FloatingActionButton(
                 onClick = onCreateNewsClick,
-                containerColor = DefaultButton
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/screens/info/news/NewsScreen.kt:103]: [See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).

File: app/src/main/java/no/usn/mob3000/ui/theme/Color.kt [hunk 32]
```diff
@@ -3,25 +3,20 @@ package no.usn.mob3000.ui.theme
...
+val OnDefaultBackground = Color(0xFF121212)
 val NavbarBackground = Color(0xFFA67F5A)
 val NavbarButtonSelected = Color(0xFF7F563B)
 val DefaultButton = Color(0XFFC0A080)
-/* TODO: Should probably make this and the DefaultButton color more easily differentiable. */
-val DefaultListItemBackground = Color(0xFFDAB892)
-val ProfileUserBackground = Color(0xFFA1887F)
-val ProfileUserStatisticsBackground = Color(0xFFEFEBE9)
+//val DefaultListItemBackground = Color(0xFFDAB892)
+//val ProfileUserBackground = Color(0xFFE0E0E0)
```
Comment (frigvid) [inline app/src/main/java/no/usn/mob3000/ui/theme/Color.kt:11]: These two are in use and should not be commented out.

File: gradle/libs.versions.toml [hunk 33]
```diff
@@ -33,6 +33,7 @@ navigation-compose = { group = "androidx.navigation", name = "navigation-compose
 androidx-security-crypto = { group = "androidx.security", name = "security-crypto", version.ref = "security-crypto" }
 kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json", version.ref = "kotlinx-serialization" }
 
+
```
Comment (frigvid) [inline gradle/libs.versions.toml:36]: Remember spacing. Do not introduce previously non-existent spacing issues, please.

**Review by frigvid — CHANGES_REQUESTED** [review]
This is not ready for merging towards master. This branch needs to be up-to-date with master before it can be approved. Additionally, it requires some changes.

I've gone through and added a good few comments, but keep in mind that some places I've neglected to comment because it'd only be repeating earlier comments verbatim. I do in some places, but simply keep in mind that comments may be applicable in multiple areas.

**Comment by Kveggjeter** [comment]
The changes proposed are just some extra lines containing "theme" and "color". The files are obviously not meant to be merged in its entirity into the structure as it is, as Markus mentioned:

_"Not merged with master, but only merged files of interest are theme refactoring in most of the screen classes"_

I see no problem merging one line into the code, instead of Markus having to copy/paste his one/two line change into all the screens when we could simply merge one line and discard the rest.
